lib/localeList.js

```diff
@@ -0,0 +1,38 @@
+"use strict";
+
+var fs = require('fs');
+var path = require('path');
+var BB = require('bluebird');
+
+var readdir = BB.promisify(fs.readdir);
+
+module.exports = function getLocaleList(root, cb) {
```

Comment:
Some comments in this function would be helpful. Maybe just a jsdoc style inputs/outputs/brief description block